skll/config.py

```diff
@@ -99,7 +99,7 @@ def __init__(self):
                                    'min_feature_count': 'Tuning',
                                    'models': 'Output',
                                    'num_cv_folds': 'Input',
-                                   'objective': 'Tuning',
```

Comment:
Having objective in `correct_section_mapping` will also need it in `defaults`, else `assert defaults.keys() == correct_section_mapping.keys()` will fail. 

Adding `objective` in `defaults` and `correct_section_mapping`, `super(SKLLConfigParser, self).__init__(defaults=defaults)` will add `objective` default value, and this will overwrite `objectives` from the logic I implemented. So, I could not find how I can do it. Suggestions please.